README.md

```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
     <img width="640px" src="screenshots/app7.png"/>
 </p>
 
+## Branches
+
+- `stable` corresponds to the latest Wazuh app stable version.
+- `master` branch contains the latest code, be aware of possible bugs on this branch.
+
+## Requisites
+
+- Wazuh HIDS 4.5.1
```

Comment:
**nitpick**: change to `4.6.0`

Reply:
Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.